tools/ios

```diff
@@ -41,6 +41,7 @@ archive_path="${rootdir}"/ios/build/ZulipMobile.xcarchive
 
 do_build() {
     yarn
+    pod install --project-directory=ios
```

Comment:
Hmm, I suggested this above, but: I'd forgotten that in this script, the working directory is `ios/`! See the `cd` command above.

Probably this is a sign that I should revise this script to use the repo root as its working directory, same as our other scripts. :slightly_smiling_face: For the moment, I'll just fix this up while merging.

Reply:
Ah, right, makes sense!